dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -148,116 +148,112 @@ const TableWithFavorite = () => {
-                onChange={handleButtonClick}
-                className="favoriteListButton"
-                aria-label="see favorites button"
-              />
-            </ToggleGroup>
-            <div className="table-scroll-container">
-              <OuterScrollContainer>
-                <InnerScrollContainer>
-                  <TableComposable
-                    aria-label="Favoritable table"
-                    variant="compact"
-                    isStickyHeader
-                  >
-                    <Thead>
-                      <Tr>
-                        <Th sort={getSortParams(0)}>{columnNames.name}</Th>
-                        <Th sort={getSortParams(1)}>
-                          {columnNames.creationDate}
-                        </Th>
-                        <Th sort={getSortParams(2)}></Th>
-                      </Tr>
-                    </Thead>
-                    <Tbody>
-                      {selectedArray.length > 0 ? (
-                        selectedArray.map((repo, rowIndex) => (
-                          <Tr key={rowIndex}>
-                            <Td
-                              dataLabel={columnNames.name}
-                              onClick={() =>
-                                navigate(`${TOC}/${repo.resource_id}`)
-                              }
-                            >
-                              {repo.name}
-                            </Td>
-                            <Td dataLabel={columnNames.creationDate}>
-                              {repo.metadata[DATASET_CREATED]}
-                            </Td>
-                            <Td
-                              favorites={{
-                                isFavorited: isRepoFavorited(repo),
-                                onFavorite: (_event, isFavoriting) => {
-                                  markRepoFavorited(repo, isFavoriting);
-                                },
-                                rowIndex,
-                              }}
-                            />
-                          </Tr>
-                        ))
-                      ) : (
-                        <Tr>
-                          <Td colSpan={8}>
-                            <EmptyTable />
-                          </Td>
-                        </Tr>
-                      )}
-                    </Tbody>
-                  </TableComposable>
-                </InnerScrollContainer>
-              </OuterScrollContainer>
-            </div>
-          </div>
-          <TablePagination
-            numberOfRows={
-              isSelected === "datasetListButton"
-                ? tableData.length
-                : favoriteRepoNames.length
-            }
-            page={page}
-            setPage={setPage}
-            perPage={perPage}
-            setPerPage={setPerPage}
-          />
         </>
       )}
+      <div className="table-container">
```

Comment:
It seems like you can simplify (new) lines 141-152, now.  Would something like this work?
```
      {!loginDetails?.isLoggedIn && loginHintVisible && (
        <LoginHint
          message="Want to see your own data?"
          link="Login or Create an account"
          onCloseMethod={onCloseLoginHint}
          redirect={APP_ROUTES.AUTH}
        />
      )}
```
That gets rid of four lines and two levels of indentation.

Reply:
Updated!